weed/server/master_server.go

```diff
@@ -372,8 +372,26 @@ func (ms *MasterServer) OnPeerUpdate(update *master_pb.ClusterNodeUpdate, startF
 	} else if isLeader {
 		go func(peerName string) {
 			raftServerRemovalTimeAfter := time.After(RaftServerRemovalTime)
+			raftServerPingTicker := time.NewTicker(5 * time.Minute)
+			defer func() {
+				ms.onPeerUpdateDoneCnExist = false
```

Comment:
this variable may be overwritten by other goroutines, if updates happen quick enough.

Seems need a better design here.

Reply:
I don't know which one is better :)